frontend/packages/process-editor/src/contexts/BpmnConfigPanelContext.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, screen, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {
+  BpmnConfigPanelFormContextProvider,
+  useBpmnConfigPanelFormContext,
+} from './BpmnConfigPanelContext';
+
+describe('BpmnConfigPanelContext', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should render children', () => {
+    render(
+      <BpmnConfigPanelFormContextProvider>
+        <button>My button</button>
+      </BpmnConfigPanelFormContextProvider>,
+    );
+
+    expect(screen.getByRole('button', { name: 'My button' })).toBeInTheDocument();
+  });
+
+  it('should provide a useBpmnConfigPanelFormContext hook', () => {
+    const TestComponent = () => {
+      const { metaDataForm } = useBpmnConfigPanelFormContext();
+      return <div data-testid='context'>{JSON.stringify(metaDataForm)}</div>;
+    };
+
+    render(
+      <BpmnConfigPanelFormContextProvider>
+        <TestComponent />
+      </BpmnConfigPanelFormContextProvider>,
+    );
+
+    expect(screen.getByTestId('context')).toHaveTextContent('');
+  });
+
+  it('should throw an error when useBpmnConfigPanelFormContext is used outside of a BpmnConfigPanelFormContextProvider', () => {
+    // Mock console error to check if it has been called
+    const consoleError = jest.spyOn(console, 'error').mockImplementation(() => {});
+    const TestComponent = () => {
+      useBpmnConfigPanelFormContext();
+      return <div data-testid='context'>Test</div>;
+    };
+
+    expect(() => render(<TestComponent />)).toThrowError(
+      'useBpmnConfigPanelFormContext must be used within a BpmnConfigPanelContextProvider',
```

Comment:
Can we set this text in const in an util or something so the test is not dependent on the actual text? 🤔

Reply:
In this particular use-case, I believe that this text is quite static _(I can't image that this text would be changed)_ so it's simpler to keep it hard coded. The code implementation would became a little more complex due to assigning the text to a variable just for the test purpose. 😊